packages/gatsby-plugin-image/src/components/layout-wrapper.tsx

```diff
@@ -1,7 +1,9 @@
-/* global SERVER */
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
+/// <reference path="../global.d.ts" />
```

Comment:
I don't know why this has regressed, but this seems to be the onyl way I can get it to include the globals in the compilation